bot/exts/help_channels/_channel.py

```diff
@@ -44,6 +45,9 @@ def is_help_forum_post(channel: discord.abc.GuildChannel) -> bool:
 
 async def _close_help_post(closed_post: discord.Thread, closing_reason: _stats.ClosingReason) -> None:
     """Close the help post and record stats."""
+    # Get Thread with updated metadata (such as the title)
+    closed_post = await pydis_core.utils.channel.get_or_fetch_channel(bot.instance, closed_post.id)
```

Comment:
```suggestion
    closed_post = await get_or_fetch_channel(bot.instance, closed_post.id)
```

Looks like `get_or_fetch_channel` is already imported in this file, so we can drop the `import pydis_core` entirely